db/seeds.rb

```diff
@@ -1,7 +1,16 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+require 'date'
+
+User.create!(email: "marco@lewagon.com", password: "123456")
+User.create!(email: "ebi@lewagon.com", password: "123456")
+
+Category.create!(name: "Chores", user_id: 1)
+
+Day.create!(name: "Su's special day", highlight: "Workout", created_at: DateTime.parse("2023-06-06 12:00"), user_id: 1)
+Task.create!(description: "Be nice to Louis", priority: 1, day_id: 1)
+Task.create!(description: "Laundry", day_id: 1, category_id: 1)
+Task.create!(description: "Learn to code finally", day_id: 1)
+
+Day.create!(highlight: "Painting class", created_at: DateTime.parse("2023-02-05 12:00"), user_id: 1)
```

Comment:
This line shows that a day doesnt need a name and it will be defaulted to "today"